portfolio/src/main/webapp/index.html

```diff
@@ -1,14 +1,24 @@
 <!DOCTYPE html>
 <html>
+
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+
+  <body bgcolor="#ff9999">
+    <h1 class="title">Megha Kak</h1>
+    <div class="top-navigation"> 
+        <a href="index.html">Home</a>
+        <a href="hobbies.html">Hobbies</a>
+        <a href="gallery.html">Gallery</a>
+        <a href="blog.html">STEP Blog</a>
+    </div>
+    <hr color=black>
+
     <div id="content">
-      <h1>Megha's Portfolio</h1>
       <p>Thank you for checking out my portfolio! My name is Megha and I'm a rising junior at UC Irvine majoring in 
         Computer Science and Engineering. I like to be involved on campus, to make my family and friends laugh, 
         and to put together music playlists for my friends! 
```

Comment:
Sorry I missed this last time but could you add an alt attribute to the image below as well. And remove the `/` from the img end tag as it is not needed

https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback